servers/src/main/java/tachyon/master/MasterBase.java

```diff
@@ -155,6 +166,16 @@ public void stop() throws IOException {
         mStandbyJournalTailer.shutdownAndJoin();
       }
     }
+    mExecutorService.shutdownNow();
```

Comment:
@gpang What do you think of creating the threadpool in start() instead of in the constructor?